django/contrib/admin/templatetags/admin_static.py

```diff
@@ -1,17 +1,17 @@
-from django.apps import apps
+import warnings
+
 from django.template import Library
+from django.templatetags.static import static as _static
+from django.utils.deprecation import RemovedInDjango20Warning
 
 register = Library()
 
-_static = None
+warnings.warn('"admin_static" template tags will be'
```

Comment:
I think you'll need to move this into the function itself, otherwise it looks like the warning will trigger even if you don't use the template tag (that seems to be the reason for the build failure).